app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def new; end
```

Comment:
- [ ] Why did you put `new` instead of `show`. Kindly change it to the correct one.